dev/Dockerfile.silverstripe

```diff
@@ -153,59 +122,87 @@ RUN \
     # Enables apache modules
     a2enmod headers rewrite ssl
 
+####
+# Runtime user/project/directory setup
+####
+
+RUN \
+    # Changes user id of www-data to 1000 for permissions and shares
+    # compatibility with other machines
+    usermod -u 1000 www-data; \
+    groupmod -g 1000 www-data; \
+    # Removes files we won't use from /var/www
+    rm -rvf /var/www/*; \
+    # Gives ownership of /var/solr to www-data to allow index creation
+    mkdir /var/solr && chown -R www-data:www-data /var/solr; \
+    # Gives ownership of npm tmp dir to www-data
+    mkdir /var/www/.npm && chown -R www-data:www-data /var/www/.npm; \
+    # Gives ownership of composer tmp dir to www-data to allow index creation
+    mkdir -p /var/www/.composer/cache && chown -R www-data:www-data /var/www/.composer
+
+# Install SSPak and Composer
+RUN \
+    # Installs SSPAK for SilverStripe
+    # https://github.com/silverstripe/sspak
+    curl -sS https://silverstripe.github.io/sspak/install | php -- /usr/local/bin && \
+    # Composer binary
+    curl -sS https://getcomposer.org/installer | MEMORY_LIMIT=-1 php -- --filename=composer --install-dir=/usr/local/bin && \
```

Comment:
Could you lock this to a specific version of composer? (by adding `--1` or `--2` depending on what version we're using)